src/app/Caches/CacheMetrics.tsx

```diff
@@ -186,16 +187,16 @@ const CacheMetrics = (props: { cacheName: string; display: boolean }) => {
           <TextContent>
             <TextList component={TextListVariants.dl}>
               <TextListItem component={TextListItemVariants.dt}>
-                {displayUtils.formatNumber(stats.current_number_of_entries)}
+                {displayUtils.formatNumber(size)}
               </TextListItem>
               <TextListItem component={TextListItemVariants.dd}>
-                Current number
+                Size
               </TextListItem>
               <TextListItem component={TextListItemVariants.dt}>
-                {displayUtils.formatNumber(stats.total_number_of_entries)}
+                {displayUtils.formatNumber(stats.required_minimum_number_of_nodes)}
               </TextListItem>
               <TextListItem component={TextListItemVariants.dd}>
-                Total number
+                Required Minimum No of nodes
```

Comment:
@oraNod Which is the best label for this value?

Reply:
I think "Minimum number of nodes" sounds fine.